org.eclipse.xtext.xbase.lib.gwt/pom.xml

```diff
@@ -14,10 +14,6 @@
 	<description>The runtime library for the Xbase programming language.</description>
 
 	<dependencies>
-		<dependency>
```

Comment:
isnt this transitive dependency?

Reply:
Right!

![image](https://user-images.githubusercontent.com/1202254/215452428-92fa3881-81d5-4ecf-b435-32b33d97d4c0.png)

And, slightly OT, IIRC, one that gives a lot of headaches in testing due to mismatch in signatures with javax.annotations. When testing with mockito for example. ;)